cf/polyfills.js

```diff
@@ -81,19 +89,23 @@ export const fs = {
 }
 
 export const net = {
-  isIP: x => RegExp.prototype.test.call(IPv4Reg, x) ? 4 : RegExp.prototype.test.call(IPv6Reg, x) ? 6 : 0,
+  isIP: (x) =>
+    RegExp.prototype.test.call(IPv4Reg, x)
```

Comment:
Curious, why not use the simpler `IPv4Reg.test(x)` ?

Reply:
Straight up copy from the deno node polyfill 🙈 Might be something about their contexts and the RegExp instance used, but that shouldn't apply to us, so yes, let's change it to the simpler method :)